stac_generator/point_stac_generator.py

```diff
@@ -0,0 +1,352 @@
+import datetime as pydatetime
+from collections.abc import Sequence
+from itertools import chain
+from pathlib import Path
+from typing import Any, Literal, cast
+from uuid import UUID, uuid4
+
+import numpy as np
+import pandas as pd
+import pystac
+from pystac.extensions.projection import ItemProjectionExtension
+
+from stac_generator.generator import StacGenerator
+from stac_generator.typing import (
+    BBoxT,
+    CSVMediaType,
+    DateTimeT,
+    FrameT,
+    GeometryObj,
+    PointBandInfo,
+    TimeExtentT,
+)
+
+
+# Helper functions
+def ensure_name_suffix(name: str, suffix: str) -> str:
+    """Ensure that name ends with suffix.
```

Comment:
Sorry to be an absolute pain but could we use reST style docstrings in MCCN repos? Does this conflict with mkdocs?

"""
This is a reST style.

:param param1: this is a first param
:param param2: this is a second param
:returns: this is a description of what is returned
:raises keyError: raises an exception
"""

Reply:
Shouldn't affect mkdocs, but I will need to find a way to configure vscode docstring extension. I will give it a try